internal/dataplane/kongstate/kongstate_test.go

```diff
@@ -819,7 +819,7 @@ func BenchmarkGetPluginRelations(b *testing.B) {
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
-		fr := ks.getPluginRelations(store, logr.Discard())
+		fr := ks.getPluginRelations(store, logr.Discard(), nil)
```

Comment:
Can we provide the resource failure collector here? Even if it's not used I believe it would help testing `getPluginRelations` more comprehensively. WDYT?